tasks/main.yml

```diff
@@ -122,6 +131,26 @@
   with_items:
     - pacemaker
     - corosync
+  when: not cluster_is_remote | bool
+
+- name: Get cluster status
+  command: pcs cluster status
+  run_once: true
+  changed_when: false
+  register: cluster_status
+
+- name: Add remote node
+  vars:
+    cluster_hostname: "{{ hostvars[inventory_hostname][cluster_hostname_fact] }}"
+    delegate_host: "{{ hostvars[groups['cluster_is_remote_False'][0]].inventory_hostname }}"
+    # NOTE: Without this, the host's ansible_host variable will not be
+    # respected when using delegate_to.
+    ansible_host: "{{ hostvars[delegate_host].ansible_host | default(delegate_host) }}"
```

Comment:
- [ ] - on my testing system (`ansible 2.9.10`) with this line (containing `ansible_host`) I always receive following error
~~~
TASK [ondrejhome.ha-cluster-pacemaker : Add remote node] **********************
skipping: [192.168.22.54]
skipping: [192.168.22.55]
fatal: [192.168.22.56]: FAILED! => {"msg": "'hostvars' is undefined"}
~~~
Commenting it out works well for me on `ansible 2.9.10`. Which error you see on `ansible 2.8` when this is not present?

Reply:
Here is a simplified version of the setup I use that requires this:

```
my_host ansible_host={{ my_ip }}

host_vars/my_host:
my_ip: 1.2.3.4
```

I've been using this pattern for quite a long time, with various versions of Ansible.

I wonder why you get an error about hostvars? It should be available

Reply:
I will check with this pattern and let you know. Thank you for sharing an example.